app/controllers/granules_controller.rb

```diff
@@ -42,7 +42,10 @@ def timeline
 
   class GranuleUrlStreamer
     def initialize(params, token, url_mapper, echo_client, url_type=:download)
-      params.reject!{|p| ['datasource', 'short_name'].include? p}
+      @is_cwic = params['datasource'].present? && params['datasource'] != 'cmr'
```

Comment:
This should just be params['datasource'] == 'cwic'. As-is, this prevents us from having other data sources